### ISSUE TITLE
Limit number of signatures in sc.yaml

### ISSUE THREAD

**Issue by Dmdv** [0chain/0chain#838]
```
type MintPayload struct {
	EthereumTxnID     string                 `json:"ethereum_txn_id"`
	Amount            state.Balance          `json:"amount"`
	Nonce             int64                  `json:"nonce"`
	Signatures        []*AuthorizerSignature `json:"signatures"`
	ReceivingClientID string                 `json:"receiving_client_id"`
}

```

**Comment by ma2b0043** [comment]
@Dmdv when can we expect to close this issue?

**Comment by boddumanohar** [comment]
~currently blocked by : https://github.com/0chain/0chain/issues/1602~

~so fixing that issue first.~

**Comment by peterlimg** [comment]
closed via #1667
